_editing/5_front_matter.md

```diff
@@ -222,6 +222,8 @@ styles:
 {% endraw %}
 {% endhighlight %}
 
+Don't forget the "s" character at the end of your array name.
```

Comment:
Okay !
So, I read the part of the documentation again and I didn't saw the line where you clarify the singularity.
Sorry, my bad !